docs/content/docs/connectors/datastream/formats/azure_table_storage.md

```diff
@@ -0,0 +1,130 @@
+---
+title:  "Microsoft Azure table"
+weight: 4
+type: docs
+aliases:
+- /dev/connectors/formats/azure_table_storage.html
+- /apis/streaming/connectors/formats/azure_table_storage.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Microsoft Azure Table Storage format
+
+_Note: This example works starting from Flink 0.6-incubating_
```

Comment:
I don't think we need to include this note, since we don't support Flink 0.6 anymore (and the documentation is specifically targeted towards Flink 1.15)

Reply:
agree, it was on dataSet conncetor page (cf my general comment) I did not clean anything. Will do thx for pointing out